File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        x = 0

        count = 0

        while i < len(nums):
            if nums[i] > nums[x]:
                x += 1
                nums[x] = nums[i]
                count += 1
            
            i += 1

        return count + 1

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.ts
function removeDuplicates(nums: number[]): number {
    let left = 1
    let right = 1

    for (let i = 1; i < nums.length; i++) {
        if (nums[right] != nums[right-1] ) {
            nums[left] = nums[right]
            left += 1
        }
        right += 1
    }

    return left
};